src/components/atoms/LinkListItem/LinkListItem.tsx

```diff
@@ -0,0 +1,33 @@
+import Image from "next/image"
+import Link from "next/link"
+
+export const LINK_TYPE = {
+  YOUTUBE: "youtube",
+  TWITTER: "twitter",
+  INSTAGRAM: "instagram",
+  OTHER: "website_link",
+} as const
+
+type LINK_TYPE = (typeof LINK_TYPE)[keyof typeof LINK_TYPE]
+type LinkListItemProps = Required<{
+  platform: LINK_TYPE
+  linkTitle: string
+  linkDiscription: string
+  linkDestination: string
+}>
+
+export const LinkListItem = ({ platform, linkTitle, linkDiscription, linkDestination }: LinkListItemProps) => {
+  return (
+    <Link href={linkDestination}>
+      <div className="w-[390px] h-16 px-4 py-3 border border-Mauve-06 justify-start items-center gap-4 inline-flex">
+        <Image src={`/${platform}_icon.svg`} width={36} height={36} alt={`${platform}Icon`} />
+        <div className="grow flex-col justify-start items-start gap-1 inline-flex">
+          <div className="text-Mauve-12 text-fs16 font-Inter">{linkTitle}</div>
+          {linkDiscription && <div className="text-Mauve-11 text-fs14 font-Inter">{linkDiscription}</div>}
```

Comment:
Youtubeなどで説明文を表示することはなさそうなので、
説明文の表示条件に`platform === LINK_TYPE["OTHER"]`入れると、linkDescription を Optionalにしたときの人為的なミスをTSで防げそうと思いました。
```suggestion
          {platform === LINK_TYPE["OTHER"] && linkDiscription && <div className="text-Mauve-11 text-fs14 font-Inter">{linkDiscription}</div>}
```

Reply:
ありがとうございます！
指摘の通り、より厳密に表示条件をチェックするように修正しました！